compiler/onnx-tools/onnx-dump.py

```diff
@@ -43,6 +43,28 @@ def _get_attribute_value(attr):
     return helper.get_attribute_value(attr)
 
 
+def _gather_value_infos(onnx_model):
+    vis = dict()
+
+    for mod_input in onnx_model.graph.input:
+        vis[mod_input.name] = mod_input.type
+
+    for mod_output in onnx_model.graph.output:
+        vis[mod_output.name] = mod_output.type
+
+    for vi in onnx_model.graph.value_info:
+        vis[vi.name] = vi.type
+
+    return vis
+
+
+def _type_format(type):
+    dtstr = _data_type_str(type.tensor_type.elem_type)
+    shape = type.tensor_type.shape
+    shape_ar = [-1 if dim.dim_value == 0 else dim.dim_value for dim in shape.dim]
```

Comment:
Do you have any reference that changes 0 to -1?

Reply:
It is my preference from tflite :)
As talked off-line, we can go as default `0` for ONNX.